templates/deployment.yaml

```diff
@@ -41,10 +41,20 @@ spec:
       serviceAccountName: {{ include "n8n.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      {{- if .Values.initContainers }}
+      {{- if or .Values.initContainers (and .Values.persistence.enabled (eq .Values.persistence.type "dynamic")) }}
       initContainers:
+      {{- if and .Values.persistence.enabled (eq .Values.persistence.type "dynamic") }}
+      - name: init-data-dir
+        image: busybox:1.31.1
```

Comment:
have you not been able to use the n8n image?
The reason is we might need to bump this busybox version regularly as vulnerability scanner will complain about the issue. It is just a matter of time if someone complains the version is not changeable.